### PR TITLE
Display images using flex box

### PR DESCRIPTION
**Pull request by haoyxc**
Add images to the food section of index and corresponding css to display in a flex grid. This made the total content in index greater than 100vh so I made the background and header not move on scroll (only the actual content will scroll). 

Here's a look at that section
image

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,27 +1,62 @@
...
+
+/* BACKGROUND IMAGES STYLES */
+#body {
+  background-position: center;
+  background-image: url(/images/background/sunset3.jpg);
+  background-attachment: fixed;
+  min-height: 100%;
+  background-size: cover;
+  background-repeat: no-repeat;
+  /* height: 100vh;
```
Comment (EatingW) [inline portfolio/src/main/webapp/style.css]: nit: delete?

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -51,32 +86,35 @@
...
-li:last-child {
-  flex-grow: 10;
-}
-li > img {
-  max-height: 100%;
-  min-width: 100%;
-  object-fit: cover;
-  vertical-align: bottom;
+  flex-direction: row; 
+  padding: 0 4px
```
Comment (EatingW) [inline portfolio/src/main/webapp/style.css]: nit: missing trailing ;

**Review by EatingW — COMMENTED** [review]
LGTM!

**Review by fenghaolw — COMMENTED** [review]
LGTM in general, but I also have the same question: is it safe to include all these images :)

Let me follow up with intern team (and cc you) to see if there are any legal restrictions on this.

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (haoyxc) [reply portfolio/src/main/webapp/style.css]: deleted!

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (haoyxc) [reply portfolio/src/main/webapp/style.css]: fixed!

**Review by fenghaolw — APPROVED** [review]

**Review by fenghaolw — CHANGES_REQUESTED** [review]
Remove sunset3.jpg? Seems like it is from a old version of #5.

**Comment by haoyxc** [comment]
I just pulled master, should be fixed now!

**Review by fenghaolw — APPROVED** [review]

**Pull request merged**